src/include/configuration/configuration.h

```diff
@@ -10,9 +10,24 @@
 //
 //===----------------------------------------------------------------------===//
 
-#pragma once
+// internal use only in
+// configuration/configuration_manager.cpp
+#ifdef __PELOTON_CONFIG__
 
-#include <gflags/gflags.h>
+#include "configuration/configuration_utils.h"
```

Comment:
This file should be called "configuration_util.h" to match the other files in the repository.